elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -37,14 +37,12 @@ class PfeAccordion extends PFElement {
       oldDisclosure: {
         type: String,
         alias: "disclosure",
-        attr: "pfe-disclosure",
-        cascade: ["pfe-accordion-header", "pfe-accordion-panel"]
+        attr: "pfe-disclosure"
```

Comment:
we don't need the cascade for the old prefixes?

Reply:
No alias'ed attributes need to cascade, no.  They are copied to the new version and the new one cascades.